SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialRPCService.cpp

```diff
@@ -255,6 +255,12 @@ void SpatialRPCService::ExtractRPCsForEntity(Worker_EntityId EntityId, Worker_Co
 void SpatialRPCService::OnCheckoutEntity(Worker_EntityId EntityId)
 {
 	const MulticastRPCs* Component = View->GetComponentData<MulticastRPCs>(EntityId);
+	if (Component == nullptr)
```

Comment:
This fixes an issue where entities can now be checked out by the load balancing worker with only the ACL and auth intent checked out, as that is all it needs. I am assuming the worker does not need to deal with multicast RPCs in that case as the only work it should be doing is setting the ACL correctly.

Reply:
Side note, it is very difficult to exhaustively test all the combinations of properties that could break when changing something as fundamental as interest

Reply:
This function is now called on addition of the multicast RPC component instead of the entity checkout